apps/pie-docs/src/assets/styles/components/_homePage.scss

```diff
@@ -0,0 +1,110 @@
+@use '../fozzie-config' as f;
```

Comment:
I think this should be named `_hero.scss`. If that's already taken, then a more specific name :)